Automattic/wp-calypso
# Add: Ability to download backups for expired Atomic sites

Related to pIFnH-7g8-p2

## Proposed Changes

* Currently there is not a way for users to download backups (if they exist) from their expired Atomic site. This PR will introduce a card on the `/backup/{siteSlug}/` page for simple sites that have been reverted and have a valid backup, that will allow users to download the latest valid backup.

## Why are these changes being made?


* When sites are reverted, a lossless import will trigger, however, all plugins, custom themes, theme customizations, custom post types (such as Woo products), and files stored outside of the media library are all lost, as they are not supported on simple plans. This change allows users to download a backup from the `Jetpack > Backup` page, so as long as it exists.

## Testing Instructions



* Visit `/backup/{siteSlug}/` on a simple site that has never been Atomic - only the upgrade card should appear
* Visit `/backup/{siteSlug}/` on a simple site that was Atomic and has been reverted, and has a valid backup - the download flow should appear under the upgrade card, and should display the date of the backup.
* Test downloading the backup flow

## Download Screens:

![image](https://github.com/user-attachments/assets/f2159161-ec22-4407-ba8b-b3ddb93737dd)

![image](https://github.com/user-attachments/assets/76a63332-6433-45a2-832e-84e03c9a3eb3)

![image](https://github.com/user-attachments/assets/316a627f-8644-4a23-af31-3c24388416c3)


## To-do

- [x] Make styling changes requested here: pIFnH-7g8-p2#comment-27853
- [x] ~~Add collapsible section if confirmed that this should be added to Jetpack sites as well: pIFnH-7g8-p2#comment-27855~~
- [ ] The wording for email notifications has been commented out, as currently this functionality is not supported on simple sites, as they face rewind authorization issues. We'll need to look into how feasible this is.
- [ ] We'll need to split up the existing `download.tsx` component and abstract the header/body content to make it dynamic, while keeping the download functionality itself separate, ensuring that we can remove `download-expired-plan.tsx`, as it duplicates functionality.

## Pre-merge Checklist
